authui/src/authflowv2/components/otp-input.css

```diff
@@ -63,6 +63,8 @@
       @apply caret-white/0;
       /* iOS treats element with `opacity: 0` as not interactable, set text to transparent instead */
       @apply text-transparent;
+      /* To solve iOS autofill reveals text color */
+      -webkit-text-fill-color: var(--otp-input__bg-color);
```

Comment:
```
--otp-input__dev-1511-color-transparent: transparent;

// https://linear.app/authgear/issue/DEV-1511
// When iOS autofills a code from SMS, the filled text will use -webkit-text-fill-color instead of color as text color, so we need to set both property to transparent.
// Continue to write down why simple transparent does not work. Attach the stackoverflow link here.
-webkit-text-fill-color: var(--otp-input__dev-1511-color-transparent);
```